src/server/src/test/java/com/spotify/reaper/unit/resources/ClusterResourceTest.java

```diff
@@ -31,126 +31,169 @@
     assertEquals(SEED_HOST, cluster.getSeedHosts().iterator().next());
   }
 
   @Test
   public void testAddExistingCluster() throws Exception {
+    final MockObjects mocks = initMocks();
+
     Cluster cluster = new Cluster(CLUSTER_NAME, PARTITIONER, Sets.newHashSet(SEED_HOST));
-    context.storage.addCluster(cluster);
+    mocks.context.storage.addCluster(cluster);
 
-    ClusterResource clusterResource = new ClusterResource(context);
-    Response response = clusterResource.addCluster(uriInfo, Optional.of(SEED_HOST));
+    ClusterResource clusterResource = new ClusterResource(mocks.context);
+    Response response = clusterResource.addCluster(mocks.uriInfo, Optional.of(SEED_HOST));
     assertEquals(403, response.getStatus());
     assertTrue(response.getEntity() instanceof String);
     String msg = response.getEntity().toString();
     assertTrue(msg.contains("already exists"));
-    assertEquals(1, context.storage.getClusters().size());
+    assertEquals(1, mocks.context.storage.getClusters().size());
   }
 
   @Test
   public void testGetNonExistingCluster() throws ReaperException {
-    ClusterResource clusterResource = new ClusterResource(context);
-    Response response = clusterResource.getCluster("i_dont_exist", Optional.<Integer>absent());
+    final MockObjects mocks = initMocks();
+
+    ClusterResource clusterResource = new ClusterResource(mocks.context);
+    Response response = clusterResource.getCluster(I_DONT_EXIST, Optional.<Integer>absent());
     assertEquals(404, response.getStatus());
   }
 
   @Test
   public void testGetExistingCluster() throws ReaperException {
-    final String I_DO_EXIST = "i_do_exist";
+    final MockObjects mocks = initMocks();
+
     Cluster cluster = new Cluster(I_DO_EXIST, PARTITIONER, Sets.newHashSet(SEED_HOST));
-    context.storage.addCluster(cluster);
+    mocks.context.storage.addCluster(cluster);
 
-    ClusterResource clusterResource = new ClusterResource(context);
+    ClusterResource clusterResource = new ClusterResource(mocks.context);
     Response response = clusterResource.getCluster(I_DO_EXIST, Optional.<Integer>absent());
     assertEquals(200, response.getStatus());
   }
 
   @Test
   public void testModifyClusterSeeds() throws ReaperException {
-    ClusterResource clusterResource = new ClusterResource(context);
-    clusterResource.addCluster(uriInfo, Optional.of(SEED_HOST));
+    final MockObjects mocks = initMocks();
+
+    ClusterResource clusterResource = new ClusterResource(mocks.context);
+    clusterResource.addCluster(mocks.uriInfo, Optional.of(SEED_HOST));
 
-    when(jmxProxy.getLiveNodes()).thenReturn(Arrays.asList(SEED_HOST+1));
-    
-    Response response = clusterResource.modifyClusterSeed(uriInfo, CLUSTER_NAME,
-        Optional.of(SEED_HOST + 1));
+    when(mocks.jmxProxy.getLiveNodes()).thenReturn(Arrays.asList(SEED_HOST + 1));
+
+    Response response =
+        clusterResource.modifyClusterSeed(mocks.uriInfo, CLUSTER_NAME, Optional.of(SEED_HOST + 1));
```

Comment:
fyi: google style will request that the operator `=` be the start of the new line.

this doesn't need to be fixed here, with a big format commit around the corner.